.github/workflows/basic.yml

```diff
@@ -133,6 +67,15 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y clang cmake libnuma-dev libjemalloc-dev libtbb-dev
 
+    - name: Install oneAPI basekit
+      if: matrix.compiler.cxx == 'icpx'
+      run: |
+        sudo apt-get install -y gpg-agent wget
+        wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+        echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+        sudo apt-get updat
```

Comment:
heh, you have a misspell here, that's why it was failing on your fork, I guess 😉